ci/prow/perf_config.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// data definitions that are used for the config file generation of performance
+// tests cluster maintanence jobs.
+
+package main
+
+import (
+	"fmt"
+	"log"
+)
+
+// generatePerfClusterUpdatePeriodicJobs generates periodic jobs to update clusters
+// that run performance testing benchmarks
+func generatePerfClusterUpdatePeriodicJobs() {
+	for _, repo := range repositories {
+		if repo.EnablePerformanceTests {
+			perfClusterPeriodicJob(
```

Comment:
nit: this could be a single function call with just 2 different parameters. But definitely not a blocker.

Reply:
I didn't make changes to this one since I feel it more verbose to add an extra function here.